Salary calculation and vacations (#36)
EPIC-guide/epic-guide.github.io: _middle/vacations.md

---
layout: post
title: "Vacations"
category: middle
order: 4
---

The information on this page has largely been translated from the laws in French
(also available in German and Italian), so in case of any discrepancy, the
official Swiss languages take precedence.

## Personal leave

As an EPFL employee, you are entitled to five weeks of paid personal leave every
calendar year, and an additional week if you are under 20 or over 50. You must

> take two consecutive weeks of leave every year, and you must use up all five

Reviewer: ```suggestion
take two consecutive weeks of leave every year, you must use up all five
```

weeks in each calendar year as far as possible, and EPFL HR really does not want
you to carry vacation days over year to year.

Various circumstances can additionally count as paid time off (see sources), of
which the following are probably most relevant to PhD students:
* your own marriage: 6 days
* the marriage of a family member: 1 day
* first aid or medical treatment of a member of your household: up to 3 days in each instance
* if you are moving house: 1 day per calendar year
* the death of a parent or member of your household: 5 days

> * the death of an other family member: up to 3 days

Reviewer: ```suggestion
* the death of another family member: up to 3 days
```

course, paid leave ends if the contract ends, which is usually within a year for
PhD students.

([RS 172.220.113 arts. 36–36c](https://www.fedlex.admin.ch/eli/cc/2001/279/fr#art_36))

## Parenthood

If you have newly become a mother, you are entitled to 4 months of paid
maternity leave, starting at most 1 month before the birth of the child. You can
choose to convert the second half of your leave into reduced working hours with
the agreement of your supervisor, e.g. 2 months at 50% instead of 1 month of
full leave. Also, talk to your doctoral program head for details about stopping
the ‘PhD clock’.

If you have newly become a father, you are entitled to 20 days of paid paternity
leave, of which 10 must be taken in the first six months after birth and the
other 10 within the first year. Additionally, if both parents work in the EPF
domain, the reduced working hours mentioned above can be shared between them.

There are other provisions for adoption, illness of young children, etc. Consult
the laws for more details. Sorry for the heteronormative language—that’s just
how the laws are phrased (in French) and I didn’t want to introduce inaccuracies.

> ([RS 172.220.113 arts. 47–47b](https://www.fedlex.admin.ch/eli/cc/2001/279/fr#art_47),

Reviewer: ```suggestion
([RS 172.220.113 arts. 37–37b](https://www.fedlex.admin.ch/eli/cc/2001/279/fr#art_37),
```

[Maternity – EPFL HR](https://www.epfl.ch/campus/services/human-resources/en/maternity/),
[Pregnancy, maternity, paternity, breastfeeding – EPFL Community & Support](https://www.epfl.ch/campus/community/family-and-work-life/pregnancy-maternity-paternity-breastfeeding/))

## Remote work

> No academic discussion of vacations is complete without mentioning remote work…

Reviewer: I'm a bit skeptical of this section because in practice it's much more flexible than this. I get that you wanted to stay close to the rules but IMHO the EPIC guide can afford to talk about how things are in practice, which is why it's more helpful than official EPFL pages.

I suggest rephrasing to some form of "talk to your advisor to know what they allow; if you cannot come to an agreement with them, know that..."

“Teleworking is neither a right nor an obligation” per the rules. Any remote
work policy/schedule has to be agreed upon by you and your supervisor and the
agreement can be ended with one month’s notice. “[Teleworking from abroad] is
not to exceed two weeks per year, and is subject to the requirement that the
employee must have used her or his vacation from prior years by 30 April of the
current year.” Occasional remote work does not have to be recorded with HR, but

> a regular remote work agreement does.

Reviewer: I would just drop that last sentence, it makes the process look more complex than it really is. There are plenty of other "you must inform HR" cases that nobody does in practice either.
